spectre_analyses/tasks.py

```diff
@@ -0,0 +1,140 @@
+from contextlib import contextmanager
+from functools import partial
+import json
+import os
+import pickle as pkl
```

Comment:
Why the abbreviation? Is this common?

Reply:
It was some time ago, but code got updated to just 2 usages. Removed abbreviation.